controllers/nestedcluster_controller.go

```diff
@@ -97,7 +98,8 @@ func (r *NestedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		if err := r.Status().Update(ctx, nc); err != nil {
 			return ctrl.Result{}, err
 		}
+		return ctrl.Result{}, nil
 	}
 
-	return ctrl.Result{}, nil
+	return ctrl.Result{RequeueAfter: 5 * time.Second}, nil
```

Comment:
For now this will just constant poll for changes until status provisioned, this should in the long run be triggered by the NCP status updating instead.